app/livechat/server/api/v1/contact.js

```diff
@@ -23,7 +23,8 @@ API.v1.addRoute('omnichannel/contact', { authRequired: true }, {
 			if (this.bodyParams.phone) {
```

Comment:
Why are you dealing with the `phone` field here rather than inside the `registerContact` method?